databases.go

```diff
@@ -120,6 +147,12 @@ type ClusterPrice struct {
 	Monthly float32 `json:"monthly"`
 }
 
+// DatabaseFork describes the source and restore time for the fork for forked DBs
+type DatabaseFork struct {
+	Source      int        `json:"source"`
+	RestoreTime *time.Time `json:"restore_time,omitempty"`
```

Comment:
I think we'll need to override `UnmarshalJSON` for DatabaseFork like we do for the database structs to properly parse the time strings 👍

Reply:
Done!